Dockerfile

```diff
@@ -7,13 +7,13 @@ ADD collectd.conf /etc/collectd/
 ADD entrypoint.sh /entrypoint.sh
 
 RUN echo 'ef97838e249814ad96a2edc0410326efbb4f3d08  collectd-5.5.0.tar.bz2' > /tmp/collectd.sig && \
-    apt-get update && \
-    apt-get install -y build-essential wget tar lbzip2 && \
+    apt-get update && apt-get install -y build-essential wget tar lbzip2 && \
```

Comment:
Yep. The way docker containers create the underlying file system is to basically store the diff between Dockerfile commands. This means if you have a lot of commands apart from each other then they are stored as differnet layers.

```
RUN apt-get update
RUN apt-get upgrade -y 
```

The above is two differnet layers, each stored with the diff of the changed fs. The goal to keep containers manageable is to have as small of an underlying fs diff as possible. This means doing lots of things in one step (and then cleaning up after). 

http://developers.redhat.com/blog/2016/03/09/more-about-docker-images-size/